OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -438,9 +438,9 @@ public static void SetupSpawnWidget(Widget parent, Session.Slot s, Session.Clien
 		public static void SetupEditableReadyWidget(Widget parent, Session.Slot s, Session.Client c, OrderManager orderManager, MapPreview map)
 		{
 			var status = parent.Get<CheckboxWidget>("STATUS_CHECKBOX");
-			status.IsChecked = () => orderManager.LocalClient.IsReady || c.Bot != null;
+			status.IsChecked = () => orderManager.LocalClient.IsReady;
 			status.IsVisible = () => true;
```

Comment:
STATUS_CHECKBOX is defined as Visible: false in yaml and set as true here. Is there a reason for not removing both of them?